gstaichi/program/launch_context_builder.cpp

```diff
@@ -242,6 +244,25 @@ void LaunchContextBuilder::set_arg_ndarray(const std::vector<int> &arg_id,
   set_arg_ndarray_impl(arg_id, ptr, arr.shape);
 }
 
+void LaunchContextBuilder::set_arg_ndarray(const std::vector<int> &arg_ids,
```

Comment:
Can we put these things in a new PR please? I already eviewed and approved this one ,right?

Reply:
oh apparenlty I didnt approve yet. but still